core/arch/arm/plat-zynq7k/platform_config.h

```diff
@@ -212,11 +212,11 @@
 
 #define TZDRAM_BASE			(CFG_DDR_TEETZ_RESERVED_START)
 #define TZDRAM_SIZE				(CFG_DDR_TEETZ_RESERVED_SIZE - \
-						CFG_PUB_RAM_SIZE)
+							CFG_PUB_RAM_SIZE)
 
-#define CFG_SHMEM_START			(CFG_DDR_TEETZ_RESERVED_START + \
+#define TEE_SHMEM_START			(CFG_DDR_TEETZ_RESERVED_START + \
 						TZDRAM_SIZE)
-#define CFG_SHMEM_SIZE			CFG_PUB_RAM_SIZE
+#define TEE_SHMEM_SIZE			CFG_PUB_RAM_SIZE
```

Comment:
We should consider replacing `CFG_PUB_RAM_SIZE` with `CFG_SHMEM_SIZE`, doesn't have to be done in this commit though.

Reply:
ok. Whatever, this part will need some rework. There is equivalent changes on going through https://github.com/OP-TEE/optee_os/pull/2217